src/main/java/com/rk/rentacarapp/services/RentsService.java

```diff
@@ -0,0 +1,11 @@
+package com.rk.rentacarapp.services;
+
+import com.rk.rentacarapp.model.User;
+import com.rk.rentacarapp.model.Vehicle;
+
+import java.time.LocalDate;
+
+public interface RentsService {
+
+    String requestVehicleRent(Vehicle vehicle, User user, LocalDate startDay, LocalDate endDay);
```

Comment:
Consistency with empty rows. You can have an empty row at the top and the bottom, or no empty row at the top or the bottom. But it has to be consistent